### ISSUE TITLE
Selected images don't clear on IOS

### ISSUE THREAD

**Issue by tkortekaas** [Sh1d0w/multi_image_picker#548]
**Bug Report**

`multi_image_picker` version: 4.7.13

Is it happening on Android: No 

Is it happening on iOS: Yes

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.22.0-10.0.pre.157, on Mac OS X 10.15.6 19G2021,
    locale en-GB) 
[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
```

**Describe the bug**

1) Create a button (or another way to run functions) that when pressed runs "MultiImagePicker.pickImages()"
2) Press this button
3) Select an arbitrary amount of images
4) Stop the image selection
5) Press the button again, which again results in the image picker opening
6) You should see that the previously selected images are still selected. 

Basically the images you select don't clear after closing the picker. The images do disappear on Android which is (I think) the preferred behaviour. This also happens after setting "selectedAssets: []"

**Comment by eug48** [comment]
I'm seeing this too - it seems to be a side-effect of commit bcc33fbcd435ba7f442e286374e7cb8de051b4f6. Reverting it fixes this for me.

**Comment by trustmefelix** [comment]
I am facing this issue too, may I know is it fixed?

**Comment by eug48** [comment]
@trustmefelix flutter has a very nice feature that lets you refer to specific revisions of plugins in your pubspec.yaml - so you can make a fork with commit bcc33fb reverted or use mine:

```
  multi_image_picker:
    git:
      url: https://github.com/eug48/multi_image_picker.git
      ref: 0312d57301166e504fde84de1db41a7d6300ee8e
```

**Comment by trustmefelix** [comment]
@eug48  Thanks for your help

**Comment by Sh1d0w** [comment]
@eug48 Thanks for the heads up. This is now fixed in version 4.7.14 that I've just published.
